app/src/lib/stores/app-store.ts

```diff
@@ -3954,9 +3953,14 @@ export class AppStore extends TypedBaseStore<IAppState> {
     }
 
     const repository = selection.repository
+    const state = this.repositoryStateCache.get(repository)
+
     this.repositoryStateCache.update(repository, () => ({
-      conflictState: {
-        branch: tip.branch,
+      changesState: {
```

Comment:
Same thing here, this can become:

```ts
this.repositoryStateCache.updateChangesState(repository, () => ({
  conflictState: {
    branch: tip.branch,
  }
}))
```